File: Java/Lista 3/L3_Ex1.4/src/UsuarioGroup.java

public class UsuarioGroup extends Usuario{
    
    public UsuarioGroup(String login) {
        super(login);
    }
    
    public void descricaoGroup(){
        System.out.println("O usuario Group, pode add e remover usuarios de grupos");
    }
    
    public void groupBoard(){
        System.out.println(" Esse usuario Group poderá apagar usuários de seu grupo");
    }
    
    public void despedidaGroup(){
        System.out.println("Até logo" + this.getLogin() + "!");
    }
    
}

File: Swift/Lista_03_EX04/UsuarioGroup.swift
import Foundation
 
 public class UsuarioGroup : Usuario{
 
	func descricaoGroup(){
		print("O usuario Group, pode add e remover usuarios de grupos. ")
	}
	
	func groupBoard(){
		print("Esse usuario Group poderá apagar usuários de seu grupo. ")
	}
	
	func despedidaGroup( ) {
		print("Até logo \(login)!")
	}
	
}